src/sentry/integrations/base.py

```diff
@@ -451,7 +443,7 @@ def is_rate_limited_error(self, exc: Exception) -> bool:
         raise NotImplementedError
 
     @property
-    def metadata(self) -> IntegrationMetadata:
+    def metadata(self) -> dict[str, Any]:
```

Comment:
this was typed incorrectly, the `IntegrationMetadata` is not associated with each individual integration instance but rather the overall integration implementation (we list the features, the author, a description for the integration,etc). the integration instance's metadata is a dict that varies per integration